src/contracts/account/ProxyAccountDeployer.sol

```diff
@@ -92,11 +91,10 @@ contract ProxyAccount is ReplayProtection {
 
 
 /**
- * A minimal relay hub contract.
- * Verifies the signer's signature and replay protection before forwarding calldata to the target.
- * Delegates nonce verification to another contract.
+ * Responsible for deploying new proxy accounts via CREATE2
+ * Every user has their own proxy account.
  */
-contract ProxyHub is ReplayProtection {
+contract ProxyAccountDeployer is ReplayProtection {
```

Comment:
we can move this to a new ticket.

although i feel there is 1 issue. Let's say the user wants to deploy several contracts in a single transaction (like we do with any.sender). 

Anyone could front-run the transaction and create just one of them. That would make the user's transaction fail potentially (since 1 out of N contracts already existed). It may have some security implications for the .init() methods in that case.